src/ui/Views/Legal/Cookies.cshtml

```diff
@@ -0,0 +1,116 @@
+
+        <h3 class="heading-medium">Measuring website usage (Google Analytics)</h3>
+
+        <p>We use Google Analytics software to collect information about how you use ‘Publish postgraduate teacher training courses’. We do this to help make sure the site is meeting the needs of its users and to help us make improvements. </p>
+
+        <p>Google Analytics stores information about:</p>
+        <ul class="list list-bullet">
+            <li>the pages you visit on ‘Publish postgraduate teacher training courses’</li>
+            <li>how long you spend on each page</li>
+            <li>how you got to the site</li>
+            <li>what you click on while you’re visiting the site</li>
+        </ul>
+
+        <p>We don’t allow Google to use or share our analytics data.</p>
+
+        <p>Google Analytics sets the following cookies:</p>
+
+        <div class="body-text">
+            <table>
+                <thead>
+                    <tr>
+                        <th>Name</th>
+                        <th>Purpose</th>
+                        <th>Expires</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td>_ga</td>
+                        <td>Used to distinguish anonymous users</td>
+                        <td>2 years</td>
+                    </tr>
+                    <tr>
+                        <td>_gid</td>
+                        <td>Used to distinguish anonymous users</td>
+                        <td>24 hours</td>
+                    </tr>
+                    <tr>
+                        <td>_gat_gtag_UA_114739594_3</td>
+                        <td>Throttle requests</td>
+                        <td>10 minutes</td>
+                    </tr>
+                </tbody>
+            </table>
+        </div>
+
+        <p>You can opt out of <a href="https://tools.google.com/dlpage/gaoptout" rel="external">Google Analytics</a>.</p>
+
+        <h3 class="heading-medium">Our introductory message</h2>
+
+        <p>You may see a pop-up welcome message when you first visit ‘Publish postgraduate teacher training courses’. We’ll store a cookie so that your computer knows you’ve seen it and knows not to show it again.</p>
+
+        <div class="body-text">
+            <table>
+                <thead>
+                    <tr>
+                        <th>Name</th>
+                        <th>Purpose</th>
+                        <th>Expires</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td>seen_cookie_message</td>
+                        <td>Show cookie policy banner on first visit only</td>
+                        <td>1 month</td>
+                    </tr>
+                </tbody>
+            </table>
+        </div>
+
+        <p>Other ‘Publish postgraduate teacher training courses’ cookies may include:</p>
```

Comment:
This content doesn't fit under the `Our introductory message` heading. Should this `p` be an `h3`?